### ISSUE TITLE
This doesn't solve the problem

### ISSUE THREAD

**Issue by NikitaIT** [oxc-project/eslint-plugin-oxlint#6]
I had to add `reportUnusedDisableDirectives:false` to my eslint config to prevent autofix from removing comments.

```
reportUnusedDisableDirectives: false
```

In jest tests I always get a false positive `eslint(no-undef)`.

The VSCode plugin is not configurable for some of the rules listed here. This is why I don't get any hints about `typescript-eslint(no-explicit-any)`. It would be nice to separate the rules into those that are enabled by default and those that are not. Now, in order to disable something in eslint, I need to go through all the rules and check that they work. Unfortunately, right now I can't delegate part of the pipeline from eslint to oxlint for only one reason - I can't find out that oxlint is not working as expected until I run eslint.

Overall I really like the speed. But I still should turn off the rules explicitly.

**Comment by Dunqing** [comment]
Thank you for your feedback, Yes we should separate the rules because, Because in usually, people only use default rules
